packs/data/familiar-abilities.db/erudite.json

```diff
@@ -0,0 +1,38 @@
+{
+    "_id": "cy7bdQqVANipyljS",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Your familiar has incredible religious knowledge. Despite being a minion, your familiar gains 1 reaction at the start of its turns, which it can use only to Aid you on a Religion check (it still has to prepare to help you as normal for the @Compendium[pf2e.actionspf2e.Aid]{Aid} reaction). It automatically succeeds its check to Aid you with Religion checks or automatically critically succeeds if you're a master in Religion.</p>"
```

Comment:
```suggestion
            "value": "<p>Your familiar has incredible religious knowledge. Despite being a minion, your familiar gains 1 reaction at the start of its turns, which it can use only to @Compendium[pf2e.actionspf2e.Aid]{Aid} you on a Religion check (it still has to prepare to help you as normal for the Aid reaction). It automatically succeeds its check to Aid you with Religion checks or automatically critically succeeds if you're a master in Religion.</p>"
```